### PR TITLE
Issue #117 : fixed lint errors in Resources and Tools components

### PR DESCRIPTION
**Pull request by Hyunsujk**
fixed lint errors in Resources and Tools components Issue #117 

- [x] ran app

- [x] ran lint -- lint errors are gone

- [x] ran test -- test good

### CODE REVIEW

File: src/app/resources/resources.component.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export class ResourcesComponent implements OnInit {
     this.topics = TOPICS;
   }
 
-  ngOnInit() {
+  public ngOnInit(): void {
```
Comment (alisaduncan) [inline src/app/resources/resources.component.ts:16]: Yay! Thanks for adding the scope visibility! It's my preference to have them too!

File: src/app/contact/contact.component.html [hunk 2]
```diff
@@ -14,7 +14,7 @@ <h5>Questions? Concerns? Press requests?</h5>
           Tweet us.</a>
       </div>
       <div class="col-sm-5">
-        <a href="http://www.kcwomenintech.org" target="_blank" class="kcwit-logo">
+        <a href="https://kcwomenintech.org/" target="_blank" class="kcwit-logo">
```
Comment (alisaduncan) [inline src/app/contact/contact.component.html:17]: Would it be possible to remove this? I think the problem is that you need to fetch latest from upstream, merge it into your origin, and rebase your branch off of it?

**Review by alisaduncan — COMMENTED** [review]
Almost there! 1 change request as noted below.

File: src/app/contact/contact.component.html (hunk 2 above)
Comment (Hyunsujk) [reply src/app/contact/contact.component.html]: I did it!! 😭

**Review by alisaduncan — APPROVED** [review]
You beat Git! Yay!
 
Fixes #117

**Pull request merged**